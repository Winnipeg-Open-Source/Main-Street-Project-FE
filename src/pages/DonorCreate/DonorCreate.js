import React from 'react';
import Page from 'components/Page';
import DonorCreationForm from 'components/DonorCreationForm';

function DonorCreate () {
    return (
        <Page title='Create a donor'>
            <DonorCreationForm />
        </Page>
    );
}

DonorCreate.displayName = 'DonorCreate';

export default DonorCreate;
