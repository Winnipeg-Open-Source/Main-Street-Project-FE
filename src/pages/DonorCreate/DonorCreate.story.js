import React from 'react';
import DonorCreate from './DonorCreate';

export const EmptyForm = () => <DonorCreate />;

export default {
    title: 'Pages|DonorCreate',
    component: DonorCreate,
};
