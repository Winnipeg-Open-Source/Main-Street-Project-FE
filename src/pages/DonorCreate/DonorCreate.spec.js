import React from 'react';
import renderWithTheme from 'tests/helpers/renderWithTheme';
import DonorCreate from './DonorCreate';

describe ('DonorCreate', () => {
    it ('renders without crashing', () => {
        const { asFragment } = renderWithTheme(<DonorCreate />);

        expect(asFragment()).toMatchSnapshot();
    });
});
