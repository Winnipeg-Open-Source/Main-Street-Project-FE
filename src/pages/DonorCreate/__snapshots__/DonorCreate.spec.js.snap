// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonorCreate renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pbJYR hNzbLI sc-pkfPC jpeBiC"
  >
    <div
      class="sc-AxjAm sc-AxiKw bENblX"
      width="1"
    >
      <div
        class="sc-AxjAm sc-AxiKw iPzLAe"
        width="1"
      >
        <div
          class="sc-AxhCb fppuj"
          font-size="3"
        >
          Create a donor
        </div>
      </div>
      <div
        class="sc-AxjAm sc-AxiKw sc-qQxFs cDVLUs"
      >
        <div
          class="sc-AxjAm sc-AxiKw sc-qYUWV foBumL"
        >
          <input
            class="sc-pTWqp fyIUKN"
            font-size="2,,1"
            placeholder="Name"
          />
          <input
            class="sc-pTWqp fyIUKN"
            font-size="2,,1"
            placeholder="Phone Number"
          />
          <input
            class="sc-pTWqp fyIUKN"
            font-size="2,,1"
            placeholder="Email"
          />
          <input
            class="sc-pTWqp fyIUKN"
            font-size="2,,1"
            placeholder="Street Address"
          />
          <input
            class="sc-pTWqp fyIUKN"
            font-size="2,,1"
            placeholder="City"
          />
          <input
            class="sc-pTWqp fyIUKN"
            font-size="2,,1"
            placeholder="Postal Code"
          />
        </div>
        <div
          class="sc-AxjAm sc-AxiKw sc-psDhf dCBSxT"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
